scripts/morty/isobar_morty_pde.py

```diff
@@ -0,0 +1,502 @@
+        """
+        Run MORTY, calculating the isobar concentrations in-core and ex-core
+            for the given problem.
+
+        Returns
+        -------
+        result_mat : 3D matrix
+            Holds values over time, space, and nuclide (in that order)
+
+        """
+        result_mat = self._initialize_result_mat()
+        for ti, t in enumerate(ts[:-1]):
+
+            self._update_sources()
+
+            self.conc_a = self._external_PDE_no_step(self.conc_a, 'a')
+            self.conc_b = self._external_PDE_no_step(self.conc_b, 'b')
+            self.conc_c = self._external_PDE_no_step(self.conc_c, 'c')
+            self.conc_d_m1 = self._external_PDE_no_step(self.conc_d_m1, 'd_m1')
+            self.conc_d = self._external_PDE_no_step(self.conc_d, 'd')
+
+            result_mat = self._update_result_mat(result_mat, ti)
+
+        return result_mat
+
+    def _external_ODE_no_step(self, conc, isotope, t):
+        """
+        This function applies a single time step iteration of the ODE
+
+        Parameters
+        ----------
+        conc : float
+            Initial concentration
+        isotope : string
+            Nuclide isobar indicator (a, b, c, d, or d_m1)
+
+        Returns
+        -------
+        conc : float
+            Concentration at current time
+        """
+
+        conc = (conc * np.exp(-self.mu[isotope][0] * t) +
+                self.S[isotope][0] / self.mu[isotope][0] *
+                (1 - np.exp(-self.mu[isotope][0] * t)))
+
+        return conc
+
+    def ode_solve(self):
+        """
+        Solve the time dependent ODE
+
+        """
+        ODE_result_mat = self._initialize_result_mat(False)
+        conc0_a = self.conc_a
+        conc0_b = self.conc_b
+        conc0_c = self.conc_c
+        conc0_d = self.conc_d
+        conc0_d_m1 = self.conc_d_m1
+
+        for ti, t in enumerate(ts[:-1]):
+            self._update_sources(False)
+
+            self.conc_a = self._external_ODE_no_step(conc0_a, 'a', t)
+            self.conc_b = self._external_ODE_no_step(conc0_b, 'b', t)
+            self.conc_c = self._external_ODE_no_step(conc0_c, 'c', t)
+            self.conc_d_m1 = self._external_ODE_no_step(conc0_d_m1, 'd_m1', t)
+            self.conc_d = self._external_ODE_no_step(conc0_d, 'd', t)
+
+            ODE_result_mat = self._update_result_mat(ODE_result_mat, ti)
+
+        return ODE_result_mat
+
+    def parallel_MORTY_solve(self):
+        """
+        Run MORTY, calculating the isobar concentrations in-core and ex-core
+            for the given problem.
+
+        Returns
+        -------
+        result_mat : 3D matrix
+            Holds values over time, space, and nuclide (in that order)
+
+        """
+        import multiprocessing
+        result_mat = self._initialize_result_mat()
+        for ti, t in enumerate(ts[:-1]):
+            # Parallelization is easy due to Jacobi appraoch
+            #   Each isobar is independent from the others at each iteration
```

Comment:
This shouldn't be a comment. If you want to include it, put it in the docstring.